remo/voting/views.py

```diff
@@ -23,6 +21,7 @@ def list_votings(request):
     """List votings view."""
     user = request.user
     polls = Poll.objects.all()
+    PAST_POLLS_PER_PAGE = 10
```

Comment:
Seems kind of weird this breaks the lower case convention followed else where in the code.